src/transformers/models/blip_2/modeling_blip_2.py

```diff
@@ -1073,6 +1277,8 @@ def forward(
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
+        input_ids: Optional[torch.FloatTensor] = None,
+        position_ids: Optional[torch.LongTensor] = None,
```

Comment:
Following library patterns - `output_xxx` and `return_dict` arguments should go last. I'd suggest putting these after `query_embeds`.